Dockerfile

```diff
@@ -0,0 +1,346 @@
+### ---------------------------base--------------------------------------
+### Build the base Debian image that will be used in every other image
+FROM bitnami/minideb:buster as base
+RUN apt-get update
+RUN set -o errexit && apt-get -qq update
+RUN apt-get -qq install --no-install-recommends --no-install-suggests -y \
+    apt-transport-https \
+    ca-certificates \
+    bzip2 \
+    curl \
+    gnupg \
+    less \
+    lsb-release \
+    procps \
+    vim \
+    wget
+#END base
+
+### ---------------------------ddev-php-base--------------------------------------
+### Build ddev-php-base, which is the base for ddev-php-prod and ddev-webserver-*
+### This combines the packages and features of DDEV-Local's ddev-webserver and
+### DDEV-Live's PHP image
+### TODO: See if we want to just build with a single PHP version or as now with all of them.
```

Comment:
if that is still up for a debate, I'd be in favor of having multiple images, one for each PHP version, instead of single swiss army knife image.

Reply:
Just FYI, ddev-php-base and ddev-php-prod are *not* the "kitchen sink" - that's ddev-webserver-* (which has nginx and apache). ddev-php-prod as here just contains all of the PHP versions supported by sury (5.6-7.4), and decides on startup which one is the default PHP version.  I'm probably partial to it because it means less total downloads is only slightly larger than the single-php build. That said, I'm open to the single-php-per-image approach, but I think it will increase overall traffic for everybody. (This probably seems irrelevant to you because hosting is using only one version of PHP right now, but very soon you'll want 7.3 and 7.4, and with the image as here it's just an environment variable to set.